felix/proto/felixbackend.proto

```diff
@@ -605,7 +605,9 @@ message TunnelType {
 }
 
 message RouteUpdate {
+  // Deprecated - use RouteTypes instead.
   RouteType type = 1;
```

Comment:
Maybe remove this field (and mark IP ad reserved)?  You're no longer populating it so removing it will break code that uses it accidentally (at least when any consumers regenerate their protobuf).